cli/da-cli-admin.sh

```diff
@@ -10,14 +10,44 @@ printUsage() {
     echo "$0 check (b[lack]|w[hite]) GROUP_ID:ARTIFACT_ID:VERSION"
     echo "    Check if artifact GROUP_ID:ARTIFACT_ID:VERSION is in black or white list."
     echo ""
-    echo "$0 list (b[lack]|w[hite])"
-    echo "    List all artifacts in black or white list."
+    echo "$0 list black"
+    echo "    List all artifacts in blacklist."
     echo ""
-    echo "$0 add (b[lack]|w[hite]) GROUP_ID:ARTIFACT_ID:VERSION"
-    echo "    Add artifact GROUP_ID:ARTIFACT_ID:VERSION to black or white list"
+    echo "$0 list white [GROUP_ID:ARTIFACT_ID:VERSION]"
+    echo "    List all artifacts in whitelist."
+    echo "    You can optionally specify which GAV to list"
```

Comment:
Each artifact for whitelist has to be associated with a product. Will Update the documentation.